viewing_party/party.py

```diff
@@ -1,23 +1,116 @@
+import statistics
+from statistics import mode
+
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if None in (title, genre, rating):
+        return None
+    else:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if title != movie["title"]:
+            continue

```

Comment:
We could get away with omitting lines 26-27 completely. You could change your elif condition to an if condition and then if `movie["title"] == title` is not true, then your loop will just iterate to the next movie in user_data["watchlist"]